American Whitewater/Extensions/MKMapView+FitAll.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MKMapView+FitAll.swift
+//  American Whitewater
+//
+//  Created by David Nelson on 12/13/19.
+//  Copyright © 2019 American Whitewater. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import Foundation
+
+extension MKMapView {
+
+    /// When we call this function, we have already added the annotations to the map, and just want all of them to be displayed.
+    func fitAll() {
```

Comment:
Minor: Might be able to rename this to something like "fitExistingAnnotations" to make the comment redundant.